Examples/FriendlyFlix/README.md

```diff
@@ -0,0 +1,73 @@
+# FriendlyFlix
+
+## Introduction
+
+This quickstart is a movie tracker app to demonstrate the use of Firebase Data Connect
+ with a Cloud SQL database.
+
+For more information about Firebase Data Connect visit [the docs](https://firebase.google.com/docs/data-connect/).
+
+## Getting Started
+
+Follow these steps to get up and running with Firebase Data Connect. For more detailed instructions,
+check out the [official documentation](https://firebase.google.com/docs/data-connect/quickstart).
+
+### 0. Prerequisites
+- Latest version of [Xcode](https://developer.apple.com/xcode/)
+- Latest version of [Visual Studio Code](https://code.visualstudio.com/)
+- The [Firebase Data Connect VS Code Extension](https://marketplace.visualstudio.com/items?itemName=GoogleCloudTools.firebase-dataconnect-vscode)
+
+### 1. Connect to your Firebase project
+
+1. If you haven't already, create a Firebase project.
+    * In the [Firebase console](https://console.firebase.google.com), click
+        **Add project**, then follow the on-screen instructions.
+
+2. Upgrade your project to the Blaze plan. This lets you create a Cloud SQL
```

Comment:
Maybe mark this step optional since the following steps are using local emulator which doesn't requires the plan.

We can mention: 
You will need blaze plan when you deploy your schema to cloud later.

Reply:
I've restructured the README to allow for a fully local experience first, and then upgrade to Blaze once you've run the app at least once locally.